src/NServiceBus.RavenDB.Tests/Timeouts/When_adding_timeouts.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.RavenDB.Tests.Timeouts
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.Extensibility;
+    using NServiceBus.TimeoutPersisters.RavenDB;
+    using NUnit.Framework;
+    using TimeoutData = NServiceBus.Timeout.Core.TimeoutData;
+
+    public class When_adding_timeouts : RavenDBPersistenceTestBase
+    {
+        [Test]
+        public async Task Add_WhenNoIdProvided_ShouldSetDbGeneratedTimeoutId()
```

Comment:
We should just make sure, that the one adding the timeout knows the id of that timeout by either setting it himself or by getting the id as a result. The question then is if it's a TryAdd or an Add, but I guess that's stuff for a different PR anyway.